apps/homepage/src/app/[lang]/_shared/header/header.tsx

```diff
@@ -14,17 +14,25 @@ type HeaderProps = {
 };
 
 export function Header({ loc }: HeaderProps) {
+  const pathname = usePathname();
+  
   return (
     <>
       <UiViewport criteria="m|l|xl">
         <div className={styles.main} data-testid="header">
           <h1>@1<span className={styles.spanText}>x</span>Dev</h1>
           <div className={styles.buttonMenu}>
-            {buttons.map((button, index) => (
-              <IconLink key={index} href={button.href} icon={button.icon as UiIconProps["icon"]}>
-                {loc[button.key]}
-              </IconLink>
-            ))}
+            {buttons.map((button, index) => {
```

Comment:
I would move this to another component something like `MenuNavbar` that can be used inside the `UiViewport` for desktop